warehouse/forklift/legacy.py

```diff
@@ -1205,12 +1211,22 @@ def file_upload(request):
                         HTTPBadRequest,
                         "File too large. "
                         + "Limit for project {name!r} is {limit} MB. ".format(
-                            name=project.name, limit=file_size_limit // (1024 * 1024)
+                            name=project.name, limit=file_size_limit // ONE_MB
                         )
                         + "See "
                         + request.help_url(_anchor="file-size-limit")
                         + " for more information.",
                     )
+                if file_size + project.total_size > project_size_limit:
+                    raise _exc_with_message(
+                        HTTPBadRequest,
+                        "Project size too large. Limit for "
+                        + "project {name!r} total size is {limit} GB. ".format(
+                            name=project.name, limit=project_size_limit // (ONE_GB),
```

Comment:
```suggestion
                            name=project.name, limit=project_size_limit // ONE_GB,
```

Reply:
Thanks for the feedback, I implemented your recommendations in the latest commit.